org.freedesktop.LinuxAudio.Plugins.Cardinal.yml

```
@@ -0,0 +1,56 @@
+id: org.freedesktop.LinuxAudio.Plugins.Cardinal
+runtime: org.freedesktop.LinuxAudio.BaseExtension
+sdk: org.freedesktop.Sdk//21.08
+runtime-version: '21.08'
+build-extension: true
+appstream-compose: false
+branch: '21.08'
+build-options:
+  prefix: /app/extensions/Plugins/Cardinal
+  prepend-pkg-config-path: /app/extensions/Plugins/Cardinal/lib/pkgconfig
+  prepend-ld-library-path: /app/extensions/Plugins/Cardinal/lib
+modules:
+  - name: jansson
+    buildsystem: cmake-ninja
+    sources:
+      - type: archive
+        url: https://github.com/akheron/jansson/releases/download/v2.14/jansson-2.14.tar.bz2
+        sha256: fba956f27c6ae56ce6dfd52fbf9d20254aad42821f74fa52f83957625294afb9
+  - name: Cardinal
+    buildsystem: simple
+    build-commands:
+      - 'make PREFIX=${FLATPAK_DEST} SYSDEPS=true -j $FLATPAK_BUILDER_N_JOBS'
+      - 'make PREFIX=${FLATPAK_DEST} install'
+    post-install:
+      - 'install -d ${FLATPAK_DEST}/lv2/Cardinal.lv2'
+      - 'install -d ${FLATPAK_DEST}/lv2/CardinalFX.lv2'
+      - 'install -d ${FLATPAK_DEST}/lv2/CardinalSynth.lv2'
+      - 'install -d ${FLATPAK_DEST}/lxvst/CardinalFX.vst'
+      - 'install -d ${FLATPAK_DEST}/lxvst/CardinalSynth.vst'
+      - 'install -d ${FLATPAK_DEST}/vst3/Cardinal.vst3/Contents'
+      - 'install -d ${FLATPAK_DEST}/vst3/CardinalFX.vst3/Contents'
+      - 'install -d ${FLATPAK_DEST}/vst3/CardinalSynth.vst3/Contents'
+      - 'install -m 644 ${FLATPAK_DEST}/lib/lv2/Cardinal.lv2/*.*      ${FLATPAK_DEST}/lv2/Cardinal.lv2/'
+      - 'install -m 644 ${FLATPAK_DEST}/lib/lv2/CardinalFX.lv2/*.*    ${FLATPAK_DEST}/lv2/CardinalFX.lv2/'
+      - 'install -m 644 ${FLATPAK_DEST}/lib/lv2/CardinalSynth.lv2/*.* ${FLATPAK_DEST}/lv2/CardinalSynth.lv2/'
+      - 'install -m 644 ${FLATPAK_DEST}/lib/lxvst/CardinalFX.vst/*.*    ${FLATPAK_DEST}/lxvst/CardinalFX.vst/'
+      - 'install -m 644 ${FLATPAK_DEST}/lib/lxvst/CardinalSynth.vst/*.* ${FLATPAK_DEST}/lxvst/CardinalSynth.vst/'
```

```suggestion
      - 'install -m 644 ${FLATPAK_DEST}/lib/vst/CardinalFX.vst/*.*    ${FLATPAK_DEST}/lxvst/CardinalFX.vst/'
      - 'install -m 644 ${FLATPAK_DEST}/lib/vst/CardinalSynth.vst/*.* ${FLATPAK_DEST}/lxvst/CardinalSynth.vst/'
```

But I would just symlink or move.